### PR TITLE
[tiles] Add "data version" of tiles

### PR DESCRIPTION
**Pull request by chejennifer**
- for regular block tiles, add the data version where the tiles will have a "tile-data" div that holds csv data & in the case of ranking tile, will also have the svg data. 

Note: Disaster Event type tiles will need a little more work

### CODE REVIEW

**Review by shifucun — APPROVED** [review]

File: static/js/components/tiles/bar_tile.tsx [hunk 1]
```diff
@@ -107,6 +110,12 @@ export function BarTile(props: BarTilePropType): JSX.Element {
       }
       isInitialLoading={_.isNull(barChartData)}
     >
+      {props.isDataTile && barChartData && (
+        <div
+          className={DATA_CSS_CLASS}
+          data-csv={dataGroupsToCsv(barChartData.dataGroup)}
```
[the remark this answers is not in the document]
Comment (pradh) [reply static/js/components/tiles/bar_tile.tsx:116]: For now plan is to propagate back the CSV as string to LLM, so this works.

**Review by pradh — COMMENTED** [review]
Very nice!

**Review by pradh — APPROVED** [review]

File: static/js/components/tiles/bar_tile.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (chejennifer) [reply static/js/components/tiles/bar_tile.tsx:116]: I think we said we wanted the csv during our meeting? If anything, we can change this as needs change

[the remark this answers is not in the document]
Comment (shifucun) [reply static/js/components/tiles/bar_tile.tsx:116]: got it. i was not aware of the csv requirements.

**Comment by chejennifer** [comment]
thanks for the reviews!

**Pull request merged**